packs/astral/pack.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name : astral
+description : triggers for sunrise/sunset information
+version : 0.1
```

Comment:
Version should be a valid semver string (i.e. `0.1.0`). That's a recent change, that's why you can still see `X.Y` versions in old packs, but there's a PR to fix this, and in all new packs we should use valid versioning from the start.